### PR TITLE
BGDIINF_SB-1409: logging middleware

### PR DESCRIPTION
**Pull request by boecklic**
log every incoming request and response with all headers etc.

### CODE REVIEW

File: app/config/logging-cfg-local.yml [hunk 1]
```diff
@@ -42,6 +42,7 @@ filters:
     exclude_keys:
       - request.headers.Authorization
       - request.headers.Proxy-Authorization
+      - request.headers.Cookie
```
Comment (ltshb) [inline app/config/logging-cfg-local.yml:45]: This needs also to be  added to other logging configuration

File: app/config/logging-cfg-local.yml [hunk 2]
```diff
@@ -68,7 +69,7 @@ formatters:
 handlers:
   console:
     class: logging.StreamHandler
-    formatter: standard
+    formatter: json
```
Comment (ltshb) [inline app/config/logging-cfg-local.yml:72]: Please keep the local to use standard, the other two configurations uses json format.

**Review by ltshb — CHANGES_REQUESTED** [review]

**Comment by ltshb** [comment]
Nice middleware, only issues with the logging configuration. For local development we should keep as default the standard formater as it is easier to read from console. The drawback of standard formater is it can't print the extra but we should add this functionality in the logging-utilities then.

File: app/config/logging-cfg-local.yml (hunk 2 above)
Comment (boecklic) [reply app/config/logging-cfg-local.yml]: reverted

File: app/config/logging-cfg-local.yml (hunk 1 above)
Comment (boecklic) [reply app/config/logging-cfg-local.yml:45]: done

**Review by ltshb — APPROVED** [review]
:+1:

**Pull request merged**